### PR TITLE
Register Other Org feature branch for sprint 31.1

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### Change description ###

https://tools.hmcts.net/jira/browse/EUI-8694
https://tools.hmcts.net/jira/browse/EUI-8695
https://tools.hmcts.net/jira/browse/EUI-8704
https://tools.hmcts.net/jira/browse/EUI-8713
https://tools.hmcts.net/jira/browse/EUI-8714
https://tools.hmcts.net/jira/browse/EUI-8724
https://tools.hmcts.net/jira/browse/EUI-8727
https://tools.hmcts.net/jira/browse/EUI-8733
https://tools.hmcts.net/jira/browse/EUI-8752
https://tools.hmcts.net/jira/browse/EUI-8762

### Change description ###
Register Other Org feature branch for sprint 31.1
This branch is to be merged to master at the end of this sprint 31.1

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
2 minor import ordering issues

File: src/register/register.module.ts [hunk 1]
```diff
@@ -18,11 +18,13 @@ import * as fromServices from './services';
 import { HttpClientModule } from '@angular/common/http';
 import { HmctsFormBuilderModule } from './containers/hmcts-form-builder/src/lib/hmcts-form-builder.module';
 import { effects, reducers } from './store';
+import { AddressService, ExuiCommonLibModule } from '@hmcts/rpx-xui-common-lib';
```
[the remark this answers is not in the document]
Comment (johnbenjamin-hmcts) [reply src/register/register.module.ts]: I have updated the code now.

File: api/routes.ts [hunk 2]
```diff
@@ -21,9 +22,18 @@ import getTermsAndConditions from './termsAndConditions';
 import userDetailsRouter from './user';
 import getUserDetails from './user-details';
 import getUserList from './userList';
+import { showFeature } from './configuration';
```
[the remark this answers is not in the document]
Comment (johnbenjamin-hmcts) [reply api/routes.ts]: I have updated the code now.

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

File: src/register-org/components/regulatory-organisation-type/regulatory-organisation-type.component.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  public showRegulatorName: boolean;
+  public showAddRegulatorButton: boolean;
+
+  constructor(private readonly lovRefDataService: LovRefDataService,
+      public readonly router: Router,
+      public readonly registerOrgService: RegisterOrgService) {
+    super(router, registerOrgService);
+  }
+
+  ngOnInit(): void {
```
Comment (Daniel-Lam) [inline src/register-org/components/regulatory-organisation-type/regulatory-organisation-type.component.ts:28]: @johnbenjamin-hmcts - missing `public` access modifier

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - just one very minor observation

**Review by Daniel-Lam — APPROVED** [review]

**Pull request merged**